plot-doc/plot/scribblings/renderer2d.scrbl

```diff
@@ -330,6 +332,26 @@ For example, to plot an estimated density of the triangle distribution:
 @history[#:changed "7.9" "Added support for pictures for #:label"]
 }
 
+
+@defproc[(arrows [vs  (sequence/c (sequence/c #:min-count 2 real?))]
```

Comment:
This is intentional in that the original PR didn't provide this option, and I was hesitant about starting to change the "put-arrow" methods of the different area% classes.
But I think it is the way to go, yes. I was not completely happy that for each draw the parameter needs to be fetched again. (Since I heard that this can be slow)